packages/bitski-provider/src/constants.ts

```diff
@@ -107,6 +107,8 @@ export const DEFAULT_CHAINS: EthChainDefinitionWithRpcUrl[] = [
   Goerli,
   Polygon,
   Mumbai,
+  Arbitrum,
```

Comment:
Should we rename this to Arbitrum One? There's also Nova, so might be good to distinguish